classroom.rb

```diff
@@ -4,10 +4,11 @@ class Classroom
 
   def initialize(label)
     @label = label
-    @student = []
+    @students = []
   end
 
   def add_student(student)
-    @students.push(student) unless @students.include?(student)
+    @students.push(student)
+    student.classroom = self
```

Comment:
- [ ] I am not able to create a student by choosing 1 it is throwing an error. this can't arrow me to test other things like `rentals` etc
![st](https://user-images.githubusercontent.com/44745855/193407252-5299c0e2-d4e6-4259-b4f3-7a5d66819ebe.png)

Reply:
fixed